### ISSUE TITLE
[Accessibility]A11y_Java Tooling for IntelliJ/Eclipse_ Available Software_ Screen Reader: Narrator is reading “Document” instead of reading the text field value “Review the items to be installed” when focus lands on that field.

### ISSUE THREAD

**Issue by Accessibility56** [microsoft/azure-tools-for-java#3378]
[
Check out Accessibility Insights!](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-tagott%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847261333&sdata=PJMiKjb64uwhBN98LZpctnBiAVeOvw%2BLBrnxBtrjl2M%3D&reserved=0) - Identify accessibility bugs before check-in and make bug fixing faster and easier

GitHub Tags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_Java Tooling for IntelliJ/Eclipse-Stay Healthy; #MAS1.3.1; #Screen Reader; #Narrator; #Java tooling for IntelliJ/Eclipse; #C+E-H
 
**Environment Details:**
Eclipse IDE for Enterprise Java Developers.
Version: 2019-06 (4.12.0)
Build id: 20190614-1200
Solution Installed: The Azure Toolkit for Eclipse 3.17.0
 
OS Details: 
OS: Windows (Version 1903, Build: 18362.175)
Screen Reader: Narrator, NVDA
**MAS Violated**: MAS1.3.1
 **NOTE:** Same issue is observed in “Install Details and Review Licenses” screen as well.

**Repro Steps:** 
1) Start Screen Reader and 0pen Java EE - Eclipse application.
2) Tab till Help-->Install New software.
3)  Enter http://dl.microsoft.com/eclipse/ in 'Work with' text box and press enter.
4)Fill all the details and press enter on “Next” button.
5)Verify how screen reader is reading when the focus is on the first text field after Install details.

**Actual Result:** 
Narrator is reading “Document” instead of reading the text field value “Review the items to be installed” when focus lands on that field.

 **Expected Result:** 
Narrator should not  “Document” instead of reading the text field value “Review the items to be installed” when focus lands on that field.

 **User Impact:** 
Screen Reader users get confused if screen reader doesn’t give correct information.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
[Bug_3378.pptx](https://github.com/microsoft/azure-tools-for-java/files/3460716/Bug_3378.pptx)

**Comment by Accessibility56** [comment]
Logged an External Bug with IntelliJ team https://youtrack.jetbrains.com/issue/IDEA-222655

**Comment by Accessibility56** [comment]
External Bug is logged: https://bugs.eclipse.org/bugs/show_bug.cgi?id=551098

**Comment by Flanker32** [comment]
Discussed this bug with Ryan and Sarah and approved to close this issue as its external and there is non blocking scenario

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749796
